third_party/workspace.bzl

```diff
@@ -138,3 +138,12 @@ def tensorboard_workspace(name = ""):
         ],
         licenses = ["notice"],  # Apache 2.0
     )
+
+    java_import_external(
+        name = "org_jsoup_external",
+        jar_sha256 = "436adf71fe9f326e04fe134cd2785b261f0f4b9b60876adda1de3b6919463394",
+        jar_urls = [
+            "https://jsoup.org/packages/jsoup-1.21.1.jar",
+        ],
+        licenses = ["reciprocal"],  # BSD 3-clause, ICU, MPL 1.1, libpng (BSD/MIT-like), Academic Free License v. 2.0, BSD 2-clause, MIT
```

Comment:
https://jsoup.org/license says that it is distributed under the MIT license, and according to [this documentation](https://github.com/bazelbuild/bazel/blob/1daff767732b740eef96bce042e8eda9d3975fc6/tools/build_defs/repo/java.bzl#L75), MIT license should be marked as "notice".

Reply:
Good catch, I believe I've addressed this correctly